contracts/Super1155.sol

```diff
@@ -289,44 +289,15 @@ contract Super1155 is PermitControl, ERC165Storage, IERC1155, IERC1155MetadataUR
 
     @return The metadata URI string of the item with ID `_itemId`.
   */
-  function uri(uint256 id) external override view returns (string memory) {
-    Strings.Slice memory slice1 = metadataUri.toSlice();
-    Strings.Slice memory slice2 = metadataUri.toSlice(); 
-    string memory tokenFirst = "{";
-    string memory tokenLast = "}";
-    Strings.Slice memory firstSlice = tokenFirst.toSlice();
-    Strings.Slice memory secondSlice = tokenLast.toSlice();
-    firstSlice = Strings.beforeMatch(slice1, firstSlice);
-    secondSlice = Strings.afterMatch(slice2, secondSlice);
-    string memory first = Strings.toString(firstSlice);
-    string memory second = Strings.toString(secondSlice);
-    string memory result = string(abi.encodePacked(first, Strings.uint2str(id), second));
-    return result;
-  }
-
-  function getThisMetadataUri() external override view returns (string memory) {
+  function uri(uint256) external override view returns (string memory) {
     return metadataUri;
   }
 
 
-  function getTotalBalances(address _recepient)
-        external
-        view
-        override
-        returns (uint256) {
-          return totalBalances[_recepient];
-  }
-
-
   function _transferOwnership(address _owner) override external onlyOwner {
```

Comment:
Don't add a `_transferOwnership` function to `Super1155`. The `transferOwnership` function already exists in anything inheriting from `PermitControl`, which in turn inherits from `Ownable`. Let's add `transferOwnership` to the `IPermitControl` interface so we can achieve what you want to do (transferring in the DropFactory) through referencing our deployed address via the `IPermitControl` interface. This keeps the functions defined in `Super1155` cleaner because they will only be dealing with the actual NFT logic.